oom-frontend/src/app/core/users/users-backend.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { UsersBackendService } from './users-backend.service';
+
+describe('UsersBackendService', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533382)
```suggestion
describe("UsersBackendService", () => {
```